### DISCUSSION TITLE
Increase TS80P voltage to 15V?

### DISCUSSION THREAD

**Discussion by luknowak** [Ralim/IronOS#1378] [Q&A]
Many USB-PD chargers, for some reason, don't have a 12V profile. It's often 5, 9, 15, 20V. Would it be possible to increase max USB-PD voltage to 15V to get 15V/2A = 30W negotiated?
Looking at the schematic, I am not seeing any components which would have a problem with 15V - FET, diodes, DC-DC are all rated for 30V+.

**Comment by Ralim** [comment]
If I recall correctly, when I asked about 15V support the answer was hesitant due to the tip and connector construction for use at 3.3ish Amps.

If you wanted to try it, nothing stopping you from bumping the max voltage setting at compile time.

Generally as newer chargers all have PPS, this isnt an issue for those ones.
